src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -245,8 +204,13 @@ def setup(self, builder):
         )
 
     def on_initialize_simulants(self, pop_data):
+        initial_propensity = self.randomness.get_draw(pop_data.index).rename(
```

Comment:
I don't see where you use this.

Reply:
I update the state table to include these propensities in line 210